simulation_parameters/microbe_stage/biomes.json

```diff
@@ -3119,6 +3119,239 @@
       }
     }
   },
+  "banana": {
+    "Name": "COASTAL",
```

Comment:
Shouldn't the name be different so people who aren't super familiar with the banana legend would still be able to realize the easter egg? (something like `BANANA_BIOME`)